lib/send-html.js

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var ejs = require('ejs');
+var read = require('fs').readFileSync;
+var path = require('path');
+
+var assetDir = path.resolve(__dirname, '../views');
+var compileTemplate = {
+  // loading default template and stylesheet
+  default: loadDefaultTemplates(),
+};
+
+module.exports = sendHtml;
+
+function sendHtml(res, data, options) {
+  var toRender = {options: {}, data: data};
+  // TODO: ability to call non-default template functions from options
+  var body = compileTemplate.default(toRender);
+  sendReponse(res, body);
+}
+
+/**
+ * Compile and cache the file with the `filename` key in options
+ *
+ * @param filepath (description)
+ * @returns {Function} render function with signature fn(data);
+ */
+function compileTemplates(filepath) {
+  var options = {cache: true, filename: filepath};
+  return compileTemplate = ejs.compile(read(filepath, 'utf8'), options);
+}
+
+// loads and cache default error templates
+function loadDefaultTemplates() {
+  var defaultTemplate = path.resolve(assetDir, 'default-error.ejs');
+  var defaultCss = path.resolve(assetDir, 'styles', 'default-error.css');
```

Comment:
Is it necessary to explicitly load & compile the styles template? I would expect that `ejs` handles it automatically when you call `include`.

Also I would personally not create subdirectory for the single CSS file we have, and simply call it something like `style.css` or `error-style.css`